src/display/editor/color_picker.js

```diff
@@ -200,6 +228,10 @@ class ColorPicker {
     window.removeEventListener("pointerdown", this.#boundPointerDown);
   }
 
+  #isDropdownVisible() {
```

Comment:
Can't we make this a getter instead?
```suggestion
  get #isDropdownVisible() {
```

Reply:
Also, can't lines https://github.com/mozilla/pdf.js/pull/17539/files#diff-de230929082e66b9041a9047781523cd5c681cc196de4cb795ac47fab268001bR238-R239 be replaced with just `!this.#isDropdownVisible` now?